test/Common.cpp

```diff
@@ -207,6 +211,45 @@ bool CommonOptions::parse(int argc, char const* const* argv)
 	return true;
 }
 
+string CommonOptions::toString(vector<string> const& _selectedOptions) const
+{
+	if (!m_singleton || _selectedOptions.empty())
+		return "";
+
+	auto boolToString = [](bool value) -> string
+		{
+			return value ? "true" : "false";
+		};
```

Comment:
```suggestion
	auto boolToString = [](bool value) -> string
	{
		return value ? "true" : "false";
	};
```

But it's short enough that I'd just put it on a single line.